src/protocols/aeternity/helpers/index.ts

```diff
@@ -348,3 +350,11 @@ export function checkAddress(value: string) {
 export function checkAddressOrChannel(value: string) {
   return checkAddress(value) || isAddressValid(value, Encoding.Channel);
 }
+
+const SEED_LENGTH = 32;
```

Comment:
I suggest to move this into global constants file as this is not a "helper" and we use the same seed for all the protocols.